packages/config/config/devices/0x0090/910.json

```diff
@@ -16,7 +16,11 @@
 		},
 		{
 			"productType": "0x0003",
-			"productId": "0x0238",
+			"productId": "0x0238"
+		},
+		{
+			"productType": "0x0003",
+			"productId": "0x0000",
 			"zwaveAllianceId": [2003, 2239]
```

Comment:
Shouldn't this alliance ID be attached to the previous fingerprint?

Reply:
Yes, fixed.